srcpkgs/cryptsetup/template

```diff
@@ -27,7 +27,6 @@ post_patch() {
 	if [ "$XBPS_TARGET_LIBC" = musl ]; then
 		# Require losetup from util-linux, also failing when present
 		# TODO: 2 tests require lsblk
-		rm -f tests/compat-test
```

Comment:
This line is redundant as the `-f` option below already overwrites `tests/compat-test`.